### PR TITLE
UL&S: Remove `.showSigninV2`

### LINKED ISSUE #229: UL&S: Remove `.showSigninV2`
Remove the `.showSigninV2` storyboard segue from the storyboard and view controllers. Replace it with programmatic navigation.

Ref. #182

- [x] LoginEmailViewController
- [x] LoginPrologueViewController
- [x] enum case showSignupMethod
- [x] Login.storyboard - 2 references

### PR DESCRIPTION
**Pull request by mindgraffiti**
Fixes #229 

Ref. #182 

This PR removes all references to the segue `.showSigninV2` and programmatically navigates the user to the `SignupEmailViewController`. There is 1 area where the segue has been removed:

1. WPiOS signup

### To Test - WPiOS
1. Check out this branch
2. `rake dependencies` and ensure there are no errors
3. Build and run (there should be no errors)
4. Visit the WPiOS PR to run the changes: https://github.com/wordpress-mobile/WordPress-iOS/pull/13828

### CODE REVIEW

File: WordPressAuthenticator/Signin/LoginEmailViewController.swift [hunk 1]
```diff
@@ -511,7 +519,9 @@ open class LoginEmailViewController: LoginViewController, NUXKeyboardResponder {
 
 // MARK: - Google Sign In
 
-// LoginFacadeDelegate methods for Google Google Sign In
+/// Make Google Sign In conform to the LoginFacade protocol.
+/// The delegate calls these methods from LoginFacade.m.
+///
```
Comment (mindgraffiti) [inline WordPressAuthenticator/Signin/LoginEmailViewController.swift:524]: This is documentation I forgot to update for the previous segue removal, `.show2fa`

File: WordPressAuthenticator/Signin/LoginEmailViewController.swift [hunk 2]
```diff
@@ -522,6 +532,9 @@ extension LoginEmailViewController {
         googleExistingUserNeedsConnection(email)
     }
 
+    /// After a successful Google Sign In, this method gets called when the user
+    /// has enabled 2-factor authentication for their WordPress.com account.
+    ///
```
Comment (mindgraffiti) [inline WordPressAuthenticator/Signin/LoginEmailViewController.swift:537]: This is documentation I forgot to update for the previous segue removal, `.show2fa`

**Comment by bjtitus** [comment]
Tested with the work from https://github.com/wordpress-mobile/WordPress-iOS/pull/13828#pullrequestreview-388752645

:shipit:

**Comment by mindgraffiti** [comment]
Thank you @ScoutHarris and @bjtitus !

**Pull request merged**